components/camel-atom/src/main/resources/org/apache/camel/resume/adapter

```diff
@@ -0,0 +1 @@
+org.apache.camel.component.atom.UpdatedDateFilter
```

Comment:
You need to have ASF license header on these files too

Reply:
Thanks! I'll need to adjust the loader to filter out the license.